src/lucky/verify_accepts_format.cr

```diff
@@ -10,24 +10,24 @@ module Lucky::VerifyAcceptsFormat
     end
   end
 
-  # Set what formats the Action accepts.
+  # Set the single format that the Action accepts.
```

Comment:
Actually I'm not sure this is right. it can set multiple formats so this part can probably be reverted

Reply:
Never mind. Facepalm